### ISSUE TITLE
Quick download launches app instead of downloading in the background

### ISSUE THREAD

**Issue by Heli-o** [JunkFood02/Seal#1154]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

In the previous versions of Seal, using quick download from the share sheet in the background without even opening the "open with" dialog after download was complete. Now it both launches the app and then opens the open with dialog when download completes.

(System info in additional context/info)

### To Reproduce

1. Pick any TikTok video (I use seal for those)
2. Click on the share button and scroll to more so you can open the android normal share sheet
3. Click on seal quick download 

### Error reports

None

### Additional context

App version: 1.11.0-beta.2 (11000)
Device information: Android 12 (API 31)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2023.10.13

**Comment by Marco-9456** [comment]
I can't reproduce on my side

**Comment by Heli-o** [comment]
That's odd.
I guess I didn't add all of the info like what phone I got but I now also have it on video.

I uploaded it to Google photos (or will it did automatically so I just make it shareable)
https://photos.app.goo.gl/4HAuzshdcytojQQZ7

My phone is Asus Zenfone 7 pro if that helps with anything. I did try reinstall but I can try to install the stable and see if it's just my phone. Droidify version also had this issue (I now have the GitHub version).

**Comment by Heli-o** [comment]
That's odd.
I guess I didn't add all of the info like what phone I got but I now also have it on video.

I uploaded it to Google photos (or will it did automatically so I just make it shareable)
https://photos.app.goo.gl/4HAuzshdcytojQQZ7

My phone is Asus Zenfone 7 pro if that helps with anything. I did try reinstall but I can try to install the stable and see if it's just my phone. F-droid version also had this issue (I now have the GitHub version).

Edit: so I tried all 4 apks in release on GitHub. X86 weren't obviously compatible but I had to try. Both V7 and V8 head the issue (the armeabi and the arm64 or what are their names). Issue persisted. Tried the f-droid version again. Its still there. I disabled battery optimization for all and even turned off management of background apps for it.

**Comment by Marco-9456** [comment]
@JunkFood02

**Comment by JunkFood02** [comment]
> In the previous versions of Seal, using quick download from the share sheet in the background without even opening the "open with" dialog after download was complete. 

Which version?

**Comment by RequiemxOP** [comment]
> Ive tested versions from 1.8.0 onward. Always the Arm64-v8a version as thats what f-droid installed when I started using Seal. All versions ive tested will open themselves when using the quick download from share sheet. So i will chalk that up to my phone being the issue in that regard. BUT The open with dialog only kept showing up in version 1.10.0 and up. 1.9.2 and down didnt open any dialog. I dont know what settings in my phone could be making 1.10.0 show the open with dialog. Edit as I forgot to mention something: Ive always used the default settings when testing, just setting the battery to unrestricted.

I was using Flamingo with Gradle version is 8.3-rc-2. 
[The project is using an incompatible version (AGP 8.3.0-alpha06) of the Android Gradle plugin. Latest supported version is AGP 8.0.0]

Then upgrade to Giraffe. & 
[Minimum supported Gradle version is 8.3-rc-2. Current version is 8.0.] 
Can anyone provide your Studio Version & AGP & Gradle Version.

**Comment by Heli-o** [comment]
Sorry to revive this but I may have solved this issue so I'm here to say how in case someone else is a dumbass like me in the future.
I have filterbox installed on my phone. And I added a rule to auto dismiss seal download complete notifications after a certain time.
It seems I messed up the rule and accidentally included all seal notifications. This in turn seemed to have messed up Seal and started causing this bug.
So tl;dr: if you have this issue, check the notifications.
